### ISSUE TITLE
Flow chart in section 3.3 seems misplaced and is hard to follow

### ISSUE THREAD

**Issue by ramink** [hendricius/the-sourdough-framework#408]
I have trouble wrapping my head around this flowchart. First of all, it seems to have very little to do with the text of section 3.3. Perhaps just the second-to-last paragraph which mentions one last feeding, but otherwise, I'm confused by its inclusion here. To me, it seems like the flowchart is the start of dough preparation, it's a follow up to either the "make a starter" or "starter maintenance" procedure, not "how to determine if your starter is ready". Perhaps it could be called "activating your starter".

Secondly, the logic in the flowchart doesn't really work. If I were following it as an unintelligent robot, I would probably end up spinning endlessly in a cycle like so:

![flow chart loop](https://github.com/user-attachments/assets/d10b9f2f-e134-45aa-a471-1011cbe8ac77)

So I would suggest that a "wait 6-12 hours" bubble is needed after the feeding, but before the readiness check. And a better home is needed for the flowchart.

A different thing that summarises the starter readiness check would still be a good idea.

**Comment by cedounet** [comment]
I think you nailed it on the head.  Those flow charts are written by humans for humans not describing a state machine.

when I read “feed 6 hours before”, “ use a ratio of x”, “has it increased?” This is clear that you will wait 6 hours before checking.  No point adding an extra state.

Also there are sometimes esthetical constraints around those flowcharts for them to occupy a resonance vertical spacing (horizontal is constrained by A4 page width), be relatively symmetrical, nodes of same size-ish etc.

I did not think of your idea of having a table next to the flowchart so we can have relatively small nodes in each state and details on the side… that could be a good idea would need to see it to compare which one is more clear.

As far as location of those flowcharts I agree it could be better, please go ahead and submit a PR for that will be easier to reason about when we actually see where it lands versus where it is right now.

Thanks again for the feedback, really appreciate it.

**Comment by hendricius** [comment]
I agree. Probably a small text node on the right hand side could make this a bit clearer.

I created a PR which slightly updates the text. Lets close this issue and then once the descriptions are ready for the side we can open up a new issue/new PR.
